tests/integration/backends/test_tpp.py

```diff
@@ -943,6 +944,48 @@ def test_isaric_raw_clinical_variables(select_all):
     ]
 
 
+@register_test_for(tpp.open_prompt)
+def test_open_prompt(select_all):
+    results = select_all(
+        OpenPROMPT(
+            Patient_ID=1,
+            CodedEvent_ID="00000",
+            CodeSystemId=0,  # SNOMED CT
+            ConceptId="100000",
+            ConsultationDate="2023-01-01",
+            Consultation_ID=1,
+            NumericValue=1.0,
+        ),
+        OpenPROMPT(
+            Patient_ID=2,
+            CodedEvent_ID="00000",
+            CodeSystemId=2,  # CTV3 "Y"
```

Comment:
As above.

Reply:
That's an error in the description. Sorry!